bridetalk_project/src/components/molecule/navigation/footer/footer.css

```diff
@@ -0,0 +1,26 @@
+.footer {
+  display: flex;
+  position: fixed;
+  background-color: hsl(346, 58%, 98%);
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 50px;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+  /* Add the right font */
+  /* font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif */
+}
+
+.middle-text {
+  margin: auto;
+}
+.right-text {
+  margin: auto;
+}
+
+.left-text {
```

Comment:
You have 3 selectors doing exactly the same: `margin: auto`.
Thus, the classnames middle, right, and left are off.
Maybe a footer-text classname that styles the footer text? 
I would suggest positioning these elements through the parent instead tho.